source/sass/main.scss

```diff
@@ -428,6 +428,7 @@ button {
 
     &:hover {
       border: 2px solid darken($nsf-yellow, 10%);
```

Comment:
^ isn't related to the change you made in this PR. Feel free to open another PR to fix this or just use this one.